Neos.ContentRepository/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -63,14 +63,15 @@ additionalProperties:
 
               'type': { type: string, description: "PHP type of this property. Either simple type or fully qualified PHP class name." }
               'defaultValue': { type: any, description: "Default value of this property. Used at node creation time. Type must match specified 'type'." }
+              'position': { type: ['string', 'number', 'null'] }
```

Comment:
The would mean we have configs like this:

```
fooProp:
  type: string
  position: 'start'
  ui:
    inspector:
      position: 'end'
```

I fear this will confuse people.

Using the inspector position is also not awesome but at least would be a good fallback.

Reply:
Mh, `<property>.position` is already possible and evaluated. I just added it to the schema, as well..
But you're right.. The combination with `<property>.ui.inspector.position` is quite confusion..

OTOH the latter is only effective for the positioning within the inspector the other one is the "global" position

Reply:
Oh ok, so this seems to work too instead of ui.inspector.position. That's good to know!